src/DataStructures/InfluxDBClient.cs

```diff
@@ -409,9 +409,8 @@ public async Task<bool> PostPointAsync(string dbName, IInfluxDatapoint point)
         /// Sets Saved property on InfluxDatapoint to true to successful points
         ///<exception cref="UnauthorizedAccessException">When Influx needs authentication, and no user name password is supplied or auth fails</exception>
         ///<exception cref="HttpRequestException">all other HTTP exceptions</exception>
-        public async Task<bool> PostPointsAsync(string dbName, IEnumerable<IInfluxDatapoint> Points)
+        public async Task<bool> PostPointsAsync(string dbName, IEnumerable<IInfluxDatapoint> Points, int maxBatchSize = 255;)
```

Comment:
You have an extra semicolon in the function signature. Please remove and commit. It should compile.